static/css/devhub/new-landing/sections/my-addons.less

```diff
@@ -0,0 +1,3 @@
+.DevHub-MyAddons-copy {
+  max-width: 32em;
```

Comment:
Sounds good 👍. We can definitely quickly iterate on any of the minor details at the end once all the core bits are laid-out.